gui/src/locales/en.json

```diff
@@ -194,6 +195,8 @@
         "udx-url-required": "URL is required",
         "invalid-url": "Invalid URL",
         "s3-details": "S3 Account details",
+        "account-details": "Account details",
```

Comment:
Please use snake_case (All lowercase and spaces to be separated by _) not kebab-case

Reply:
So you mean - "account-details" is incorrect ?